airflow-core/src/airflow/ui/public/i18n/locales/en/hitl.json

```diff
@@ -6,6 +6,7 @@
       "received": "Reviewed"
     }
   },
+  "needsReview": "Needs Review",
```

Comment:
I understand that many use cases are probably "Review" kind of but the purpose of HITL is not only Approval, but also some "Decsion" or "Input Request".

As we anmed all previously "Action Required" or "Repsonse" which is neutral I'd favor not changing the terms here but keeping it neutral as "Needs Action". Also in the "Browse..." Menu we have it as "Required Actions"
```suggestion
  "<to be renamed>": "Requires Action",
```